.github/workflows/ci-build.yaml

```diff
@@ -42,6 +42,7 @@ jobs:
               - tasks.yaml
             tests: &tests
               - *common
+              - Dockerfile
```

Comment:
if we're not adding it to `common`, I think it would make more sense in `e2e-tests`, since those require the image builds. `tests` are units that just run Go tests directly, no Docker involved

Reply:
Yea looks like `e2e-tests` are the only ones that use the images (well and the image builds, but those currently run unconditionally, per #12006 "Future Work")

Reply:
updated this [myself](https://github.com/argoproj/argo-workflows/pull/13024/commits/c0d0e4e015a3bf8ad2eac6337ef0b9f0ea4a4a60) in order to get CI passing sooner rather than later, since Alan's left for the day in his timezone